documentation/general/test.md

```diff
@@ -14,3 +14,73 @@
+   of this issue is not to document the solution, merely to document the fact of
+   the flakiness and to prioritise someone picking the issue up very soon to fix
+   it. Of course, if you have additional details or ideas, proactively share
+   those in the issue description or discussion.
+1. Open a PR that skips the flaky test with a link to the issue as a comment or
+   in the explanatory note of the skip annotation. This PR should also be marked
+   with critical priority.
+
+The critical issue will be prioritised and assigned during the following week's
+prioritisation or community meeting, though anyone is free to work on flaky test
+resolution if they are able to.
+
+A
+[GitHub Discussion about flaky tests](https://github.com/WordPress/openverse/discussions/1824)
+is the original impetus for the development of this process. As will all
+Openverse processes, it should be iterated on and improved over time.
+
+### Identifying a "flaky test"
+
+A test is "flaky" if it passes or fails when we do not expect it to do so. The
+most common situation is for a test to fail when we did not expect it to fail.
+Be wary of false-positives when identifying flaky tests. Just because a given
+change does not look like it would cause a certain test to fail, double check
+that the test isn't correctly failing due to the changes. Sometimes tests do not
+have obvious relationships to every part of the code-base that affects them,
+especially integration or end-to-end tests.
+
+Identifying tests that pass when they shouldn't is _much_ harder but arguably an
+even more serious problem than false-negatives. It's one thing for a test to
+annoyingly fail when it shouldn't. It's an entirely different thing for a test
+to _pass_ and give us false-confidence that our applications are working as
+expected. False-negatives do not usually have a downstream effect on Openverse
+API or frontend consumers, for example. False-positives, on the other hand,
+usually mean that issues are caught only once they're in production and
+affecting users.
+
+### Test skip utilities
+
+Openverse uses pytest, Playwright, and jest for various different types of
+testing. Links to each library's test-skipping approach are below:
+
+- pytest:
+  <https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-skip>
+- Playwright:
+  - `test.skip`: <https://playwright.dev/docs/api/class-test#test-skip-1>
+  - `test.describe.skip`:
+    <https://playwright.dev/docs/api/class-test#test-describe-skip>
+- jest:
+  - `test.skip`/`it.skip`: <https://jestjs.io/docs/api#testskipname-fn>
+  - `describe.skip`: <https://jestjs.io/docs/api#describeskipname-fn>
+
+```{note}
+Jest and Playwright's `describe.skip` utilities should be used lightly and only if either the entire describe block is flaky or if the tests inside the block are interdependent and skipping an individual test will cause subsequent tests to incorrectly fail.
```

Comment:
Please manually wrap this line.

![Add flaky test documentation; link to it from playwright test failure comment #1975](https://user-images.githubusercontent.com/16580576/235600993-880e53a4-e2d7-49b1-b403-eae3b6206b22.png)

Reply:
It's pretty annoying to have to manually wrap anything :confused: And worse for it to be up to individual maintainers to notice that it needs it.